core/modules/image/image.module

```diff
@@ -1349,3 +1353,59 @@ function image_filter_keyword($value, $current_pixels, $new_pixels) {
   }
   return (int) $value;
 }
+
+/**
+ * Preprocess function for theme_image_style().
+ *
+ * If theme_image_style() is called without setting explicit height/width for
+ * an SVG image, attempt to set those dimensions. This helps in particular when
+ * uploading an image for the first time in a content form.
+ */
+function image_preprocess_image_style(&$variables) {
```

Comment:
My current suspicion is, that this whole function's unnecessary.

Reply:
Or was this the intended function to hold the image style dimension adaptions (scale down, scale up...)?